File: JS/um.js
const numero = [];

  for(let l = 0; l < 3; l++){
    const input = prompt("Insira um número: ")
    //confere a linha se já tem um array
    if (numero[l]) {
      numero[l].push(input)
      //se nao tem um array ele cria para add um numero no final
      //push: tem a função de add um numero no final
    } else {
      numero.push([ input ])
    }
  }

  for(let l = 0; l < 3; l++){
    for(let c = 0; c < 3; c++){ 
      console.log(numero[l][c]);
    }
  }

File: Python/um.py
numero = [[None, None, None], [None, None, None], [None, None, None]]

for l in range(0,3):
  for c in range(0, 3):
    numero[l][c] = int(input("Insira um valor: "))

for l in range(1):
  for c in range(1):
    print(numero)